lib/pgslice.rb

```diff
@@ -569,9 +580,17 @@ def settings_from_trigger(original_table, table)
         return [nil, nil] unless sql_format
         period = sql_format[0]
         field = /to_char\(NEW\.(\w+),/.match(function_def)[1]
+        cast = column_cast(table, field)
```

Comment:
FWIW, I removed this line, as this is an even older format and we want to default to `date` and update comment.